tests/conftest.py

```
@@ -2745,6 +2385,12 @@ def is_s390x_cluster(nodes_cpu_architecture):
     return nodes_cpu_architecture == S390X
 
 
+@pytest.fixture(scope="module")
+def skip_if_no_cpumanager_workers(schedulable_nodes):
+    if not any([node.labels.cpumanager == "true" for node in schedulable_nodes]):
```

Ruff (0.15.21)

[warning] Unnecessary list comprehension

Remove unnecessary comprehension

(C419)